src/hotspot/share/runtime/stubCodeGenerator.hpp

```diff
@@ -98,32 +99,29 @@ class StubCodeDesc: public CHeapObj<mtCode> {
 
 // forward declare blob and stub id enums
 
-enum StubGenBlobId : int;
-enum StubGenStubId : int;
-
 // The base class for all stub-generating code generators.
 // Provides utility functions.
 
 class StubCodeGenerator: public StackObj {
  private:
   bool _print_code;
-  StubGenBlobId _blob_id;
- protected:
+  BlobId _blob_id;
+protected:
   MacroAssembler*  _masm;
 
  public:
   StubCodeGenerator(CodeBuffer* code, bool print_code = false);
-  StubCodeGenerator(CodeBuffer* code, StubGenBlobId blob_id, bool print_code = false);
+  StubCodeGenerator(CodeBuffer* code, BlobId blob_id, bool print_code = false);
   ~StubCodeGenerator();
 
   MacroAssembler* assembler() const              { return _masm; }
-  StubGenBlobId blob_id()                        { return _blob_id; }
+  BlobId blob_id()                        { return _blob_id; }
```

Comment:
Align body of methods. May be move them to the left - I don't see why we have such big spacing.

Reply:
I believe the spacing is meant to align with the declarations that follow.

I have simply realigned the modified line with the old line to avoid unnecessary changes.